linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_disk_full_action.var

```diff
@@ -10,9 +10,10 @@ interactive: false
 
 options:
     default: single
-    email: email
```

Comment:
Why is email removed?

Reply:
> Why is email removed?

It is not mentioned in the list of actions in the man page.